.gitignore

```diff
@@ -47,6 +47,7 @@ tests/test-path-manager
 tests/test-commands
 tests/test-configuration
 tests/test-cxx-build
+tests/test-id-manager
```

Comment:
This sneaked in from PR #92.  Leave it and alone and close PR #92 instead?  Otherwise I'll regenerate a new PR without this change.